src/styles/style.scss

```diff
@@ -46,3 +46,5 @@
 @import '../components/playlist-countdown/playlist-countdown';
 @import '../components/playlist-next-screen/playlist-next-screen';
 @import '../components/picture-in-picture-overlay/picture-in-picture-overlay';
+@import '../components/share/share';
+@import '../components/copy-button/copy-button';
```

Comment:
need to export these components in src/components/index.js as well for external use

Reply:
already exposed J